src/main/java/io/fabric8/maven/docker/util/DockerFileUtil.java

```diff
@@ -119,6 +119,26 @@ public static FixedStringSearchInterpolator createInterpolator(MojoParameters pa
                 .withExpressionMarkers(delimiters[0], delimiters[1]);
     }
 
+    /**
+     * Fetch the location of docker file in case of simgle configuration
+     *
+     * @return directory containing the Dockerfile
+     */
+    public static String getDockerFileDirectory() {
+        if(checkFileExists("./Dockerfile")) {
+            return new String(".");
+        } else if(checkFileExists("src/main/docker/Dockerfile")){
+            return new String("src/main/docker");
```

Comment:
what's the difference between `new String("src/main/docker")` and `"src/main/docker"` ?

sorry, but that's an oldtimer ;-) and does not make sense at all as String is always immutable. so you can always return the plain string.